draft-ietf-quic-transport.md

```diff
@@ -2571,6 +2577,9 @@ the data limits set by its peer.  The cryptographic handshake stream, Stream 0,
 is exempt from the connection-level data limits established by MAX_DATA.  Stream
 0 is still subject to stream-level data limits and MAX_STREAM_DATA.
 
+If a STREAM frame is received for a stream that was previously specified as
+unidirectional, the connection MUST be closed with QUIC_INVALID_FRAME_DATA.
```

Comment:
I don't follow this. Surely the initiator of a unidirectional stream can send STREAM frames on it (and hence the receivin end is expected to receive frames). Are you trying to say that reception of a STREAM frame, by the initiator of the unidirectional stream, on that stream, is forbidden and should result in a closed connection?